analysis/src/python/data_analysis/template_errors/dataset_analysis.ipynb

```diff
@@ -0,0 +1,126 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Imports\n",
+    "import pandas as pd\n",
+    "\n",
+    "from data_analysis.model.column_name import StepColumns\n",
```

Comment:
Same as in the previous PR, maybe it is a good idea to use imports from root and add root to system paths like
```
root_path = os.path.abspath(os.path.join('..', '..', '..'))
if root_path not in sys.path:
    sys.path.append(root_path)
    
from analysis.src.python.data_analysis.model.column_name import StepColumns
from analysis.src.python.utils.df_utils import read_df, filter_df_by_single_value
```